### ISSUE TITLE
coredump after updating to 4.16.7 on Linux because of chat background

### ISSUE THREAD

**Issue by shakhzodkudratov** [telegramdesktop/tdesktop#27763]
### Steps to reproduce

1. update telegram to 4.16.7
2. from another device set a custom chat background
3. open chat
4. ~~PROFIT~~ **CRASH**

### Expected behaviour

when opening a chat with custom background, it should behave the same like a chat without custom background

### Actual behaviour

when opening a chat with background, telegram client freezes and closes

### Operating system

Manjaro Linux, Plasma 5.27.11

### Version of Telegram Desktop

4.16.7

### Installation source

Static binary from official website

### Logs

```shell
[2024.04.16 17:09:43] Launched version: 4016007, install beta: [FALSE], alpha: 0, debug mode: [TRUE]
[2024.04.16 17:09:43] Executable dir: /home/shakhzod/apps/Telegram/, name: Telegram
[2024.04.16 17:09:43] Initial working dir: /home/shakhzod/
[2024.04.16 17:09:43] Working dir: /home/shakhzod/.local/share/TelegramDesktop/
[2024.04.16 17:09:43] Command line: /home/shakhzod/apps/Telegram/Telegram
[2024.04.16 17:09:43] Executable path before check: /home/shakhzod/apps/Telegram/Telegram
[2024.04.16 17:09:43] Logs started
[2024.04.16 17:09:43] App ID: org.telegram.desktop._165b9b0175c49bffdc535b30aa4801cc
[2024.04.16 17:09:43] Connecting local socket to b47e0c45dead77a74585ca51d7dead9a-TelegramDesktop...
[2024.04.16 17:09:43] Socket connect error 0, starting server and app...
[2024.04.16 17:09:43] Moved logging from '/home/shakhzod/.local/share/TelegramDesktop/log_start0.txt' to '/home/shakhzod/.local/share/TelegramDesktop/log.txt'!
[2024.04.16 17:09:43] Opened '/home/shakhzod/.local/share/TelegramDesktop/tdata/working' for reading, the previous Telegram Desktop launch was not finished properly :( Crash log size: 559
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2024.04.16 17:09:43] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2024.04.16 17:09:43] Update Info: MTP is unavailable.
[2024.04.16 17:09:44] App Error: _basePath is empty in writeSettings()
[2024.04.16 17:09:44] Global devicePixelRatio: 1
[2024.04.16 17:09:44] QT_AUTO_SCREEN_SCALE_FACTOR: 0
[2024.04.16 17:09:44] Primary screen DPI: 96, Base: 96.
[2024.04.16 17:09:44] Computed screen scale: 100
[2024.04.16 17:09:44] DevicePixelRatio: 1
[2024.04.16 17:09:44] ScreenScale: 100
[2024.04.16 17:09:44] Icon theme: breeze-dark
[2024.04.16 17:09:44] Fallback icon theme:
[2024.04.16 17:09:44] Media Info: Type Playback, default: Kraken Tournament Edition Chat, list: { "Kraken Tournament Edition Chat" <Kraken Tournament Edition Chat>, "HyperX QuadCast Analog Stereo" <HyperX QuadCast Analog Stereo>, "Built-in Audio Digital Stereo (HDMI)" <Built-in Audio Digital Stereo (HDMI)> }, full list refresh: true
[2024.04.16 17:09:44] Media Info: Type Capture, default: HyperX QuadCast Analog Stereo, list: { "HyperX QuadCast Analog Stereo" <HyperX QuadCast Analog Stereo>, "Monitor of HyperX QuadCast Analog Stereo" <Monitor of HyperX QuadCast Analog Stereo>, "Monitor of Kraken Tournament Edition Chat" <Monitor of Kraken Tournament Edition Chat>, "C922 Pro Stream Webcam Analog Stereo" <C922 Pro Stream Webcam Analog Stereo>, "Monitor of Built-in Audio Digital Stereo (HDMI)" <Monitor of Built-in Audio Digital Stereo (HDMI)>, "Built-in Audio Analog Stereo" <Built-in Audio Analog Stereo> }, full list refresh: true
[2024.04.16 17:09:44] Media Info: Type Camera, default: usb-0000:00:14.0-9, list: { "C922 Pro Stream Webcam" <usb-0000:00:14.0-9> }, full list refresh: true
[2024.04.16 17:09:44] System tray available: [TRUE]
[2024.04.16 17:09:44] App Info: reading settings...
[2024.04.16 17:09:44] App Info: reading encrypted settings...
[2024.04.16 17:09:44] Lang Info: Loaded cached, keys: 6794
[2024.04.16 17:09:44] OpenAL Logging Level: 3
[2024.04.16 17:09:44] App Info: reading accounts info...
[2024.04.16 17:09:44] App Info: reading encrypted info...
[2024.04.16 17:09:44] App Info: reading map...
[2024.04.16 17:09:44] App Info: reading encrypted map...
[2024.04.16 17:09:44] App Info: reading encrypted user settings...
[2024.04.16 17:09:44] App Info: encrypted user settings read.
[2024.04.16 17:09:44] App Info: reading encrypted mtp data...
[2024.04.16 17:09:44] MTP Info: read keys, current: 5, to destroy: 0
[2024.04.16 17:09:44] Map read time: 17
[2024.04.16 17:09:44] App Info: reading encrypted mtp config...
[2024.04.16 17:09:44] App Info: reading map...
[2024.04.16 17:09:44] App Info: reading encrypted map...
[2024.04.16 17:09:44] App Info: reading encrypted user settings...
[2024.04.16 17:09:44] App Info: encrypted user settings read.
[2024.04.16 17:09:44] App Info: reading encrypted mtp data...
[2024.04.16 17:09:44] MTP Info: read keys, current: 3, to destroy: 0
[2024.04.16 17:09:44] Map read time: 0
[2024.04.16 17:09:44] App Info: reading encrypted mtp config...
[2024.04.16 17:09:44] App Info: reading map...
[2024.04.16 17:09:44] App Info: reading encrypted map...
[2024.04.16 17:09:44] App Info: reading encrypted user settings...
[2024.04.16 17:09:44] App Info: encrypted user settings read.
[2024.04.16 17:09:44] App Info: reading encrypted mtp data...
[2024.04.16 17:09:44] MTP Info: read keys, current: 3, to destroy: 0
[2024.04.16 17:09:44] Map read time: 0
[2024.04.16 17:09:44] App Info: reading encrypted mtp config...
[2024.04.16 17:09:44] QPainter::begin: Paint device returned engine == 0, type: 2
[2024.04.16 17:09:44] QWidget::render: Cannot render with an inactive painter
[2024.04.16 17:09:44] Export Info: Destroy top bar by controller removal.
[2024.04.16 17:09:44] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:44] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:44] Choosing xcb gl-integration based on following priority
 QList("xcb_glx", "xcb_egl")
[2024.04.16 17:09:44] Xcb GLX gl-integration created
[2024.04.16 17:09:44] Xcb GLX gl-integration successfully initialized
[2024.04.16 17:09:44] Requested format before FBConfig/Visual selection: QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize -1, redBufferSize 8, greenBufferSize 8, blueBufferSize 8, alphaBufferSize 8, stencilBufferSize -1, samples -1, swapBehavior QSurfaceFormat::DefaultSwapBehavior, swapInterval 0, colorSpace QColorSpace(), profile  QSurfaceFormat::NoProfile)
[2024.04.16 17:09:44] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:44] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:44] Available GLX extensions: GLX_ARB_context_flush_control GLX_ARB_create_context GLX_ARB_create_context_no_error GLX_ARB_create_context_profile GLX_ARB_create_context_robustness GLX_ARB_fbconfig_float GLX_ARB_framebuffer_sRGB GLX_ARB_get_proc_address GLX_ARB_multisample GLX_EXT_buffer_age GLX_EXT_create_context_es2_profile GLX_EXT_create_context_es_profile GLX_EXT_fbconfig_packed_float GLX_EXT_framebuffer_sRGB GLX_EXT_no_config_context GLX_EXT_swap_control GLX_EXT_swap_control_tear GLX_EXT_texture_from_pixmap GLX_EXT_visual_info GLX_EXT_visual_rating GLX_INTEL_swap_event GLX_MESA_copy_sub_buffer GLX_MESA_gl_interop GLX_MESA_query_renderer GLX_MESA_swap_control GLX_OML_sync_control GLX_SGIS_multisample GLX_SGIX_fbconfig GLX_SGIX_pbuffer GLX_SGIX_visual_select_group GLX_SGI_make_current_read GLX_SGI_swap_control GLX_SGI_video_sync
[2024.04.16 17:09:44] Got format: QSurfaceFormat(version 2.0, options QFlags<QSurfaceFormat::FormatOption>(), depthBufferSize 0, redBufferSize 8, greenBufferSize 8, blueBufferSize 8, alphaBufferSize 8, stencilBufferSize 0, samples -1, swapBehavior QSurfaceFormat::DefaultSwapBehavior, swapInterval 0, colorSpace QColorSpace(), profile  QSurfaceFormat::NoProfile)
[2024.04.16 17:09:44] OpenGL Profile: Compatibility.
[2024.04.16 17:09:44] OpenGL Renderer: Mesa Intel(R) Graphics (RPL-S)
[2024.04.16 17:09:44] OpenGL Vendor: Intel
[2024.04.16 17:09:44] OpenGL Version: 4.6 (Compatibility Profile) Mesa 24.0.2-manjaro1.1
[2024.04.16 17:09:44] OpenGL Extensions: GL_ARB_texture_env_dot3, GL_ARB_multitexture, GL_ARB_color_buffer_float, GL_ARB_conditional_render_inverted, GL_INTEL_shader_integer_functions2, GL_NV_primitive_restart, GL_NV_depth_clamp, GL_ARB_half_float_pixel, GL_AMD_seamless_cubemap_per_texture, GL_ARB_transform_feedback_instanced, GL_EXT_texture_shared_exponent, GL_MESA_shader_integer_functions, GL_APPLE_packed_pixels, GL_ARB_explicit_attrib_location, GL_ARB_base_instance, GL_ARB_post_depth_coverage, GL_ARB_shader_atomic_counter_ops, GL_ARB_depth_clamp, GL_AMD_query_buffer_object, GL_ARB_shader_draw_parameters, GL_ARB_explicit_uniform_location, GL_ATI_draw_buffers, GL_EXT_blend_minmax, GL_ARB_compute_shader, GL_EXT_texture_compression_rgtc, GL_ARB_texture_rectangle, GL_EXT_texture_edge_clamp, GL_EXT_texture_integer, GL_EXT_texture_compression_s3tc, GL_ARB_pixel_buffer_object, GL_EXT_shader_framebuffer_fetch_non_coherent, GL_AMD_multi_draw_indirect, GL_EXT_vertex_attrib_64bit, GL_ARB_texture_buffer_range, GL_ARB_transpose_matrix, GL_INTEL_performance_query, GL_ARB_shader_objects, GL_ARB_ES3_1_compatibility, GL_ARB_occlusion_query2, GL_EXT_compiled_vertex_array, GL_ARB_shading_language_420pack, GL_ARB_texture_storage_multisample, GL_EXT_debug_label, GL_NV_light_max_exponent, GL_ARB_pipeline_statistics_query, GL_ARB_copy_image, GL_ARB_shadow, GL_ARB_fragment_shader, GL_EXT_packed_float, GL_EXT_gpu_program_parameters, GL_ARB_texture_query_levels, GL_EXT_point_parameters, GL_ARB_invalidate_subdata, GL_ARB_texture_non_power_of_two, GL_EXT_texture_lod_bias, GL_EXT_texture_env_combine, GL_INGR_blend_func_separate, GL_AMD_texture_texture4, GL_NV_conditional_render, GL_ARB_draw_elements_base_vertex, GL_ARB_texture_view, GL_ARB_texture_barrier, GL_NV_fog_distance, GL_AMD_draw_buffers_blend, GL_ARB_parallel_shader_compile, GL_ARB_texture_float, GL_ARB_timer_query, GL_ARB_texture_mirrored_repeat, GL_ARB_vertex_array_object, GL_ARB_draw_buffers_blend, GL_ARB_ES3_2_compatibility, GL_ATI_fragment_shader, GL_ARB_fragment_program, GL_ARB_sampler_objects, GL_EXT_shader_samples_identical, GL_EXT_copy_texture, GL_EXT_texture_shadow_lod, GL_EXT_vertex_array_bgra, GL_EXT_shadow_funcs, GL_AMD_depth_clamp_separate, GL_ARB_texture_env_add, GL_ARB_uniform_buffer_object, GL_EXT_texture_cube_map, GL_ARB_compute_variable_group_size, GL_ARB_shader_atomic_counters, GL_ARB_gpu_shader5, GL_EXT_shader_integer_mix, GL_EXT_texture_array, GL_EXT_framebuffer_multisample, GL_ARB_multi_bind, GL_ARB_vertex_program, GL_EXT_texture_compression_latc, GL_EXT_packed_depth_stencil, GL_ARB_seamless_cubemap_per_texture, GL_EXT_packed_pixels, GL_ARB_get_texture_sub_image, GL_ARB_blend_func_extended, GL_AMD_vertex_shader_viewport_index, GL_ARB_spirv_extensions, GL_EXT_demote_to_helper_invocation, GL_EXT_texture_env_add, GL_NV_texture_rectangle, GL_ARB_program_interface_query, GL_AMD_performance_monitor, GL_EXT_texture_buffer_object, GL_ARB_window_pos, GL_ARB_compressed_texture_pixel_storage, GL_ARB_map_buffer_alignment, GL_EXT_framebuffer_object, GL_ARB_texture_swizzle, GL_ARB_texture_buffer_object, GL_EXT_provoking_vertex, GL_EXT_texture_object, GL_KHR_parallel_shader_compile, GL_NV_copy_image, GL_EXT_texture_sRGB_decode, GL_ARB_seamless_cube_map, GL_ARB_multi_draw_indirect, GL_ARB_texture_rgb10_a2ui, GL_KHR_debug, GL_ARB_transform_feedback_overflow_query, GL_ARB_texture_env_combine, GL_ARB_texture_stencil8, GL_ARB_shader_texture_image_samples, GL_EXT_framebuffer_multisample_blit_scaled, GL_ARB_internalformat_query, GL_NV_ES1_1_compatibility, GL_ARB_stencil_texturing, GL_ARB_shading_language_include, GL_SUN_multi_draw_arrays, GL_INTEL_blackhole_render, GL_ARB_draw_indirect, GL_ARB_buffer_storage, GL_EXT_multi_draw_arrays, GL_EXT_blend_func_separate, GL_EXT_framebuffer_sRGB, GL_EXT_blend_equation_separate, GL_AMD_vertex_shader_layer, GL_ARB_draw_instanced, GL_EXT_timer_query, GL_EXT_polygon_offset_clamp, GL_ARB_polygon_offset_clamp, GL_EXT_shader_framebuffer_fetch, GL_EXT_texture3D, GL_EXT_depth_bounds_test, GL_ARB_texture_compression, GL_ARB_conservative_depth, GL_ARB_provoking_vertex, GL_NV_texgen_reflection, GL_ARB_vertex_array_bgra, GL_ARB_occlusion_query, GL_EXT_semaphore_fd, GL_EXT_texture_env_dot3, GL_ARB_internalformat_query2, GL_ARB_clip_control, GL_EXT_draw_buffers2, GL_ARB_tessellation_shader, GL_EXT_transform_feedback, GL_ARB_texture_rg, GL_NV_packed_depth_stencil, GL_AMD_shader_stencil_export, GL_KHR_blend_equation_advanced, GL_EXT_EGL_image_storage, GL_EXT_texture_snorm, GL_ARB_direct_state_access, GL_ARB_depth_texture, GL_ARB_shader_storage_buffer_object, GL_ARB_vertex_attrib_binding, GL_INTEL_shader_atomic_float_minmax, GL_ARB_shading_language_packing, GL_ARB_separate_shader_objects, GL_ATI_separate_stencil, GL_MESA_texture_signed_rgba, GL_ARB_shader_clock, GL_EXT_subtexture, GL_ARB_shader_texture_lod, GL_ARB_instanced_arrays, GL_ARB_draw_buffers, GL_ARB_texture_cube_map, GL_ARB_compatibility, GL_ARB_derivative_control, GL_MESA_texture_const_bandwidth, GL_ARB_shader_subroutine, GL_ARB_point_parameters, GL_SGIS_texture_lod, GL_ARB_multisample, GL_ARB_sync, GL_ARB_shader_image_size, GL_EXT_texture_sRGB, GL_EXT_pixel_buffer_object, GL_EXT_secondary_color, GL_EXT_memory_object_fd, GL_ARB_fragment_coord_conventions, GL_EXT_semaphore, GL_ARB_shader_viewport_layer_array, GL_ARB_shader_precision, GL_ARB_vertex_type_10f_11f_11f_rev, GL_ARB_point_sprite, GL_EXT_separate_specular_color, GL_ARB_vertex_attrib_64bit, GL_ARB_texture_compression_rgtc, GL_IBM_texture_mirrored_repeat, GL_ARB_shading_language_100, GL_OES_read_format, GL_ARB_query_buffer_object, GL_EXT_texture_swizzle, GL_EXT_texture_compression_dxt1, GL_S3_s3tc, GL_ARB_shader_image_load_store, GL_SGIS_texture_edge_clamp, GL_ARB_vertex_buffer_object, GL_ARB_shader_group_vote, GL_ARB_texture_gather, GL_EXT_EGL_sync, GL_ARB_clear_buffer_object, GL_ARB_shader_stencil_export, GL_KHR_no_error, GL_ARB_get_program_binary, GL_ARB_copy_buffer, GL_ATI_texture_env_combine3, GL_ANGLE_texture_compression_dxt5, GL_KHR_robustness, GL_NV_fragment_shader_interlock, GL_ARB_texture_storage, GL_EXT_vertex_array, GL_EXT_abgr, GL_ARB_fragment_layer_viewport, GL_ARB_gpu_shader_int64, GL_ARB_debug_output, GL_NV_compute_shader_derivatives, GL_ARB_texture_mirror_clamp_to_edge, GL_ARB_ES2_compatibility, GL_EXT_memory_object, GL_ARB_texture_filter_anisotropic, GL_ATI_blend_equation_separate, GL_OES_EGL_image, GL_ARB_viewport_array, GL_IBM_rasterpos_clip, GL_AMD_shader_trinary_minmax, GL_EXT_stencil_wrap, GL_EXT_texture_rectangle, GL_ARB_enhanced_layouts, GL_ARB_vertex_shader, GL_ARB_map_buffer_range, GL_ARB_gpu_shader_fp64, GL_NV_alpha_to_coverage_dither_control, GL_SGIS_generate_mipmap, GL_ARB_half_float_vertex, GL_ARB_ES3_compatibility, GL_ARB_framebuffer_object, GL_ATI_texture_compression_3dc, GL_ARB_indirect_parameters, GL_ARB_framebuffer_no_attachments, GL_ARB_texture_cube_map_array, GL_ARB_robust_buffer_access_behavior, GL_ARB_vertex_type_2_10_10_10_rev, GL_EXT_rescale_normal, GL_ARB_fragment_program_shadow, GL_MESA_framebuffer_flip_y, GL_EXT_blend_subtract, GL_KHR_texture_compression_astc_ldr, GL_MESA_pack_invert, GL_ARB_texture_compression_bptc, GL_ARB_clear_texture, GL_EXT_draw_instanced, GL_KHR_robust_buffer_access_behavior, GL_NV_texture_env_combine4, GL_AMD_conservative_depth, GL_EXT_stencil_two_side, GL_ARB_robustness, GL_EXT_draw_range_elements, GL_EXT_gpu_shader4, GL_EXT_direct_state_access, GL_3DFX_texture_compression_FXT1, GL_ARB_depth_buffer_float, GL_ARB_gl_spirv, GL_ANGLE_texture_compression_dxt3, GL_ARB_texture_env_crossbar, GL_EXT_texture_filter_anisotropic, GL_EXT_framebuffer_blit, GL_ATI_texture_float, GL_EXT_blend_color, GL_MESA_window_pos, GL_NV_blend_square, GL_ARB_transform_feedback3, GL_ARB_framebuffer_sRGB, GL_ARB_shader_bit_encoding, GL_ARB_transform_feedback2, GL_KHR_texture_compression_astc_sliced_3d, GL_ARB_texture_multisample, GL_EXT_bgra, GL_NV_texture_barrier, GL_NV_half_float, GL_ARB_arrays_of_arrays, GL_KHR_blend_equation_advanced_coherent, GL_KHR_context_flush_control, GL_EXT_fog_coord, GL_EXT_texture, GL_AMD_pinned_memory, GL_IBM_multimode_draw_arrays, GL_SGIS_texture_border_clamp, GL_ARB_fragment_shader_interlock, GL_ARB_sample_shading, GL_ARB_texture_buffer_object_rgb32, GL_ARB_texture_query_lod, GL_EXT_texture_sRGB_R8, GL_ARB_texture_border_clamp, GL_ARB_shader_ballot, GL_ARB_cull_distance
[2024.04.16 17:09:44] OpenGL: QOpenGLContext created, version: 4.4.
[2024.04.16 17:09:44] OpenGL: [TRUE] (Window)
[2024.04.16 17:09:44] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:44] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:44] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:44] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:44] Notification daemon product name: Plasma
[2024.04.16 17:09:44] Notification daemon vendor name: KDE
[2024.04.16 17:09:44] Notification daemon version: 5.27.11
[2024.04.16 17:09:44] Notification daemon specification version: 1.2
[2024.04.16 17:09:44] Notification daemon capabilities: body, body-hyperlinks, body-markup, body-images, icon-static, actions, persistence, inline-reply, x-kde-urls, x-kde-origin-name, x-kde-display-appname, inhibitions
[2024.04.16 17:09:46] API Warning: not loaded minimal channel applied.
[2024.04.16 17:09:46] API Warning: not loaded minimal channel applied.
[2024.04.16 17:09:46] API Warning: not loaded minimal channel applied.
[2024.04.16 17:09:46] API Warning: not loaded minimal channel applied.
[2024.04.16 17:09:46] API Warning: not loaded minimal channel applied.
[2024.04.16 17:09:46] ./lib/jpegli/decode_scan.cc:535: Incomplete scan detected.
[2024.04.16 17:09:46] ./lib/jpegli/decode_scan.cc:535: Incomplete scan detected.
[2024.04.16 17:09:47] API Warning: not loaded minimal channel applied.
[2024.04.16 17:09:47] API Warning: not loaded minimal channel applied.
[2024.04.16 17:09:47] API Warning: not loaded minimal channel applied.
[2024.04.16 17:09:47] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:47] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:47] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:47] ./lib/jpegli/decode_marker.cc:108: Quantization table with index 0 not found
[2024.04.16 17:09:47] Assertion Failed! "image.format() == QImage::Format_ARGB32_Premultiplied || image.format() == QImage::Format_RGB32" chat_theme.cpp:775
```

**Comment by ilya-fedin** [comment]
#27745

**Comment by john-preston** [comment]
This should be fixed in 4.16.8 released now.
